src/loader-components/ExampleTemplate.module.scss

```
@@ -49,10 +50,30 @@
     display: flex;
     outline: none;
     padding: 0.625rem;
+
+    &.is-selected {
+      background-color: var(--terra-dev-site-example-template-selected-code-toggle-background-color, #666565);
+      color: var(--terra-dev-site-example-template-selected-code-toggle-color, #ebebeb);
+    }
+
+    &:hover {
+      background-color: var(--terra-dev-site-example-template-hover-code-toggle-background-color, #b8b8b8);
+
+      &.is-selected {
+        background-color: var(--terra-dev-site-example-template-hover-selected-code-toggle-background-color, #424141);
+      }
+    }
+  }
+
+  .item[data-focus-styles-enabled='true']:focus {
+    box-shadow: 0 0 1px 3px var(--terra-dev-site-example-template-keyboard-focus-color-box-shadow, #4cb2e9) inset;
+
+    &.is-selected {
+      box-shadow: 0 0 1px 3px var(--terra-dev-site-example-template-keyboard-focus-select-box-shadow, #66b5c7) inset;
```

Lets make the focus outline look more like side nav's

```suggestion
outline: 2px dashed var(--terra-dev-site-example-template-keyboard-focus-outline, #000);
outline-offset: -2px;
```

---

Updated [here](https://github.com/cerner/terra-dev-site/pull/281/commits/423db62d308b80ea7bb01ea4b9dc50a39d434cb7).